app/components/header_component/view.html.erb

```diff
@@ -1,8 +1,8 @@
-<% if @current_user %>
```

Comment:
This was really confusing because just from looking at this file, you may assume that its the instance variable from the view and its not actually passed into this component.  

https://github.com/alphagov/forms-admin/pull/381/files#diff-ac22ef01c268c39ce344f65c3939793ba98084978c87909988d4322090512d6bL7